index.src.html

```diff
@@ -1738,7 +1738,8 @@ <h4 id="request-siteboundcredential-with-mediation">
       Ask the user which {{Credential}} to share.
 
       Note: This behavior is vendor-specific. Guidelines for user agent
-      behavior are presented in [[#user-mediation]].
+      behavior are presented in [[#user-mediation]]. A user agent MAY offer
+      the user to enter and save new credentials in this step.
```

Comment:
It seems like this addition would be better as a statement in https://w3c.github.io/webappsec-credential-management/#user-mediated-selection itself. We're already outlining the requirements for the UI there, so noting this there as well seems clearer.